---
layout: archive
title: "<i>Curriculum Vitae</i>"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* Ph.D. in Biochemistry and Nanotechnology & Molecular Engineering (Dual Title Degree), University of Washington, Seattle, 2019
* B.S. in Chemistry and Biochemistry, College of Creative Studies, University of California, Santa Barbara, 2011
  * Minor in Music, College of Letters and Sciences, University of California, Santa Barbara, 2011


Work Experience
======
* February 2021 – current: <b>Computational Chemist I, Bioinformatics</b>
  * Encodia, Inc., San Diego, CA
  * Research: High-throughput computational chemistry, protein design, structural biology, and bioinformatics.
  * Supervisor: Dr. Devin Sullivan

* November 2019 – July 2020: <b>Scientist I, Protein and Cell Engineering</b>
  * Lyell Immunopharma, Inc., Seattle, WA
  * Research: High-throughput computational protein design, modeling, and structure prediction using Rosetta/PyRosetta software with Amazon Web Services (AWS) elastic compute cloud. Developed a python package for reproducible computational protein design.
  * Supervisors: Drs. Scott E. Boyken, Marc J. LaJoie, and Brian D. Weitzner

* June 2019 - November 2019: <b>Senior Fellow</b>
  * Institute for Protein Design, University of Washington, Seattle, WA
  * Research: Computational protein design of <i>de novo</i> fluorescent biosensors for super-resolution microscopy applications.
  * Supervisors: Dr. David Baker

* Spring 2014 - Spring 2019: <b>Graduate Research Assistant</b>
  * Institute for Protein Design, University of Washington, Seattle, WA
  * Research: Computational protein design of small-molecule fluorescent biosensors. Optimizing apixaban-binding proteins using computational modeling and directed evolution.
  * Supervisors: Drs. David Baker and James B. Hurley
  
* Winter 2014 - Spring 2014: <b>Graduate Research Assistant</b>
  * Department of Biochemistry, University of Washington, Seattle, WA
  * Research: Characterizing the functionally distinct domains of mammalian mitochondrial fusion proteins Mfn1/2.
  * Supervisor: Dr. Suzanne Hoppins

* Autumn 2013 - Winter 2013: <b>Graduate Research Assistant</b>
  * Department of Biochemistry, University of Washington, Seattle, WA
  * Research: Developing glutamate- and aspartate-sensing fluorescent reporters to probe photoreceptor metabolism.
  * Supervisors: Drs. James B. Hurley and Susan Brockerhoff

* Autumn 2011 - Summer 2013: <b>Post-baccalaureate Intramural Research Training Award (IRTA) Fellow</b>
  * National Institutes of Health, Bethesda, MD
  * Research: Developing a methodology to genetically transform mammalian mitochondria for studying structure-function relationships. Characterizing a temperature-sensitive mitochondrial DNA mutant D. melanogaster model. Testing a novel hypothesis for synthesizing photoactive mitochondria.
  * Supervisor: Dr. Hong Xu
  
* Summer 2010 - Autumn 2010: <b>Laboratory Assistant</b>
  * University of California, San Diego, Skaggs School of Pharmacy and Pharmaceutical Sciences
  * Research: Collecting live mouse tissue samples, genotyping transgenic/knockout mice, DNA extraction from human blood, DNA microchipping, and SNP detection by mass spectrometry
  * Supervisor: Dr. Kang Zhang

* Summer 2009 - Autumn 2009, & Summer 2010 - Autumn 2010: <b>Research Assistant</b>
  * University of California, San Diego, Scripps Institution of Oceanography
  * Research: Characterizing the signal transduction pathway of spontaneous bioluminescence in dinoflagellate cells using pharmacological agonists/antagonists for receptors implicated in luminescence.
  * Supervisor: Dr. Michael Latz
  
* Summer 2009 - Autumn 2009: <b>Computer Technician</b>
  * University of California, San Diego, Salk Institute for Biological Studies
  * Research: Designing and maintaining the laboratory website, and gaining experience with the scientific grant submission process.
  * Supervisor: Dr. Ursula Bellugi
  
* Autumn 2008 - Winter 2008: <b>Laboratory Assistant</b>
  * University of California, Santa Barbara, Department of Chemistry and Biochemistry
  * Research: Performing UV/Vis spectroscopy and high-performance liquid chromatography to isolate siderophores from marine bacteria.
  * Supervisor: Dr. Allison Butler

* Summer 2008 - Autumn 2008: <b>Marine Physical Laboratory Summer Internship</b>
  * University of California, San Diego, Scripps Institution of Oceanography
  * Research: Characterizing the signal transduction pathway of spontaneous bioluminescence in dinoflagellate cells
  * Supervisor: Dr. Grant Deane
  
* Autumn 2006 - Summer 2007: <b>Laboratory Assistant</b>
  * Muscle Tissue Core, Veterans Medical Research Foundation, La Jolla, CA
  * Research: Gaining experience with Coulter counting, gel electrophoresis, glucose assays, and western blot development using radiolabeled peptides, performing radiation tests, and making buffers.
  * Supervisors: Drs. Susan Phillips and Theodore P. Ciaraldi


Awards and Honors
======
* National Science Foundation Graduate Research Fellowship Program, 2014 - 2019
* Urdal Fellowship, University of Washington Department of Biochemistry, 2015
* National Institutes of Health Post-baccalaureate Intramural Research Training Award (IRTA) Fellowship, 2011 - 2013
* Merck and Company Index Award, 2011
* Phi Lambda Upsilon Award, National Honorary Chemical Society, 2011
* Burnham Scholarship for Engineering and Applied Sciences, 2007 - 2012
* Summer Institute of Mathematics and Science (SIMS) Program, 2007
* Excellence in Physics Award, SIMS, 2007
* Member of Golden Key International Honour Society, 2016 - current
* Member of the Rosetta Commons, 2014 - current
* Member of University of Washington Nanotechnology & Molecular Engineering Student Association, 2013 - 2019
* Member of University of California, Santa Barbara Chemistry Club, 2009 - 2011
* Member of NIH Mitochondrial Interest Group, Genetics Interest Group, and RNA Club, 2012 - 2013
* Member of the National Society of Collegiate Scholars, 2008 - current
* Member of the National Society of High School Scholars, 2006 - current

Skills
======
* Computer Programming Languages
  * Python
  * XML
  * JSON
  * Bash
  * HTML
  * PyMOL
  * py3Dmol
  * MMTF
  * Mathematica
  * MATLAB
  
* Research Fields of Expertise
  * Biochemistry
    * Western Blotting
    * Recombinant Protein Expression
    * Protein Purification
    * Tissue Culture
    * Transfection
  * Nanotechnology
    * Fluorescence Spectroscopy
    * Fluorescence Anisotropy
  * Molecular Engineering
    * Rosetta
    * Protein Structure Prediction
    * Protein-protein Interactions
    * Protein-ligand Interactions
    * Small Molecule Binder Design
  * Microscopy
    * Confocal Microscopy
    * Fluorescence Microscopy
    * Epi-fluorescence Microscopy
    * Light Microscopy
  * Protein Engineering
    * Biosensors
    * Flow Cytometry
    * Protein Chemistry
    * Protein Characterization
    * Protein Assays
    * Structural Biology
    * Stability Studies
  * Molecular Genetics
    * Polymerase Chain Reaction (PCR)
    * Real-Time PCR (qPCR)
    * Molecular Cloning
    * Genetics
    * Microbiology
    * Gel Electrophoresis

* Additional Research Skills
  * Scientific Writing
  * High-content Data Analysis
  * Assay Development
  * High-performance Computing


Publications
======

* <b>Jason C. Klima</b>, Robert A. Langan, Amanda L. Loshbaugh, Harry M. Winters, Anish Kejariwal, Scott E. Boyken, and Brian D. Weitzner. PyRosettaCluster: A python application programming interface for reproducible macromolecular structure prediction and design. Manuscript in preparation (2021).

* <b>Jason C. Klima</b>, Lindsey A. Doyle, Justin Daho Lee, Michael Rappleye, Lauren A. Gagnon, Min Yen Lee, Emilia P. Barros, Anastassia A. Vorobieva, Jiayi Dou, Samantha Bremner, Jacob S. Quon, Cameron M. Chow, Lauren Carter, David L. Mack, Rommie E. Amaro, Joshua C. Vaughan, Andre Berndt, Barry L. Stoddard, and David Baker. Incorporation of sensing modalities into de novo designed fluorescence-activating proteins. <i>Nature Communications</i>, <b>12</b>, 856 (2021). doi: 10.1038/s41467-020-18911-w.

* <b>Jason C. Klima</b>, Cameron M. Chow, Jiayi Dou, Anastassia A. Vorobieva, Lauren Carter, David Baker. Bacterial expression and protein purification of mini-fluorescence-activating proteins. <i>Protocol Exchange</i> (2021). doi: 10.21203/rs.3.pex-1077/v1.

* Kathy H. Le, Jared Adolf-Bryfogle, <b>Jason C. Klima</b>, Sergey Lyskov, Jason Labonte, Steven Bertolani, Shourya S. Roy Burman, Andrew Leaver-Fay, Brian Weitzner, Jack Maguire, Ramya Rangan, Matt A. Adrianowycz, Rebecca F. Alford, Aleexsan Adal, Morgan L. Nance, Rhiju Das, Roland L. Dunbrack, Jr., William Schief, Brian Kuhlman, Justin B. Siegel, Jeffrey J. Gray. PyRosetta Jupyter Notebooks teach biomolecular structure prediction and design. <i>Preprints</i> <b>2020</b>, 2020020097. doi: 10.20944/preprints202002.0097.v1.

* Julia Koehler Leman, Brian D. Weitzner, P. Douglas Renfrew, Steven M. Lewis, Rocco Moretti, Andrew M. Watkins, Vikram Khipple Mulligan, Sergey Lyskov, Jared Adolf-Bryfogle, Jason W. Labonte, Justyna Krys, <b>RosettaCommons Consortium</b>, Christopher Bystroff, William Schief, Dominik Gront, Ora Schueler-Furman, David Baker, Philip Bradley, Roland Dunbrack, Tanja Kortemme, Andrew Leaver-Fay, Charlie E. M. Strauss, Jens Meiler, Brian Kuhlman, Jeffrey J. Gray, and Richard Bonneau. Better together: Elements of successful scientific software development in a distributed collaborative community. <i>PLoS Computational Biology</i>, 16(5): e1007507 (2020). doi: 10.1371/journal.pcbi.1007507.

* <b>Jason C. Klima</b>, David Baker & James Bryant Hurley. Design of de novo mini fluorescence activating proteins as pH and calcium biosensors. Thesis, Ph.D., University of Washington, Seattle (2019).

* Scott E. Boyken, Mark A. Benhaim, Florian Busch, Mengxuan Jia, Matthew J. Bick, Heejun Choi, <b>Jason C. Klima</b>, Zibo Chen, Carl Walkey, Alexander Mileant, Aniruddha Sahasrabuddhe, Kathy Y. Wei, Edgar A. Hodge, Sarah Byron, Alfredo Quijano-Rubio, Banumathi Sankaran, Neil P. King, Jennifer Lippincott-Schwartz, Vicki H. Wysocki, Kelly K. Lee, David Baker. De novo design of tunable pH-driven conformational changes. <i>Science</i>, 364(6441):658-664 (2019). doi: 10.1126/science.aav7897. PMID: 31097662; PMCID: PMC7072037.

* Kelly Jin, <b>Jason C. Klima</b>, Grant Deane, Malcolm Dale Stokes, Michael I. Latz. Pharmacological investigation of the bioluminescence signaling pathway of the dinoflagellate <i>Lingulodinium polyedrum</i>: evidence for the role of stretch-activated ion channels. <i>Journal of Phycology</i>, 49: 733–745 (2013). doi: 10.1111/jpy.12084.


Teaching
======
* Instructor of PyRosetta Bootcamp, 2019, Institute for Protein Design, University of Washington
* Instructor of PyRosetta Code School, 2019, Rosetta Commons, Johns Hopkins University
* Teacher’s Assistant for Biochemistry 405: Autumn Quarter 2015, University of Washington
* Teacher’s Assistant for Biochemistry 441: Winter Quarter 2015, University of Washington
* Teacher’s Assistant for Biochemistry 440: Autumn Quarter 2014, University of Washington

 
Service and Leadership
======
* Pacific Science Center Life Sciences Research Weekend: 2014, 2015, 2016, 2017, 2018.
  * At the University of Washington Center for Game Science booth, I taught children and their parents how protein engineers discover cures to treat diseases, and explain how nanotechnology research at the University of Washington is advancing disease treatment options. Participating in this outreach program allowed me to translate my technical experience into casual language that boosts my local community's comprehension of nanotechnological research while inspiring young adults into science, technology, engineering and mathematics (STEM) fields through interactive workshops.
* University of Washington Engineering Discovery Days: 2014, 2015, 2016.
  * At this joint exhibit between the Molecular Engineering and Sciences Institute and the Washington Nanofabrication Facility, I and fellow University of Washington students and faculty in engineering shared our research with the local Seattle community, particularly with children in K-8 grades.
